project-service/server/server.js

```diff
@@ -0,0 +1,40 @@
+import http from 'http';
+import express from 'express';
+import morgan from 'morgan';
+import bodyParser from 'body-parser';
+import models from './models';
+
+import sprints from './routes/sprints';
+import tasks from './routes/tasks';
+
+let app = express();
+
+// app is a function handler being supplied to http server
+let server = http.Server(app);
+
+// middleware for http request logging
+app.use(morgan('dev'));
+
+// middlewares will parse incoming request bodies and populate req.body
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(bodyParser.json());
+
+// serve static files
+app.use(express.static(__dirname + '/../client'));
```

Comment:
As far as I understood from David, we won't need this in the project-service; the client (which now sits in the `frontend` folder) will have its own server to serve static assets.